api/questions/views.py

```diff
@@ -48,6 +48,62 @@ def post(self, request, id=None):
                 "error": message
             }, status=status.HTTP_404_NOT_FOUND)
 
+class QuestionEditViews(APIView):
+    """
+    class to edit question posted
+    """
+    permission_classes = [permissions.IsAuthenticated]
+
+    def put(self, request, id=None):
+        request.data['created_by'] = request.user
+        is_authenticated = request.user
+        owner = Question.objects.filter(created_by_id = is_authenticated).first()
+
+        if owner:
+            obj = Question.objects.get(id=id)
+            print(obj)
+            serializer = QuestionsSerializer(obj, data=request.data)
+            if serializer.is_valid():
+                Question.objects.filter(id=id).update(
+                    title=request.data.get('title'),
+                    body=request.data.get('body')
+                )
+                return Response(serializer.data, status=status.HTTP_201_CREATED)
+            return Response(serializer.errors)
+        return Response({
+            "error": "You are not the owner of this question"
+        }, status=status.HTTP_401_UNAUTHORIZED)
+    
+    def delete(self, request, id=None, m_id=None):
+        try:
+            request.data['created_by'] = request.user
+            is_authenticated = request.user
+            owner = Question.objects.filter(created_by_id = is_authenticated).first()
+
+            if owner:
+                meetup_id = Question.objects.filter(meetup_id=m_id).first()
+                question = Question.objects.filter(id=id).first()
+                if meetup_id:
+                    if question:
+                        question.delete()
```

Comment:
Hello,
This is a good work. What if you reduced the number of return statements per function or method? It would make your methods/functions to have a single entry and a single exit